tutorials/cheyenne/README.md

```diff
@@ -0,0 +1,185 @@
+
+# Cheyenne Tutorials
+
+Cheyenne is the supercomputer at the National Center for Atmopheric Research. The
+following tutorials are meant to aide users on Cheyenne with getting used to the
+different types of workflows that are possible with SmartSim.
+
+
+## Prerequisites
+
+Since SmartSim does not currently support the Message Passing Toolkit (MPT), Cheyenne
+users of SmartSim will need to utilize OpenMPI.
+
+The following modules were utilized to run the examples
+
+```
+
+Currently Loaded Modules:
+  1) ncarenv/1.3   2) gnu/8.3.0   3) ncarcompilers/0.5.0   4) netcdf/4.7.4   5) openmpi/4.0.5
+
+```
+
+With this environment loaded, users will need to build and install both SmartSim and
+SmartRedis through conda
```

Comment:
Strictly speaking, I don't think we actually REQUIRE conda. Might just want to say that SmartSim and SmartRedis can be installed via pip